lib/inspec/base_cli.rb

```diff
@@ -57,12 +57,16 @@ def self.exec_options
       profile_options
       option :controls, type: :array,
         desc: 'A list of controls to run. Ignore all other tests.'
+      # TODO: remove in inspec 2.0
       option :format, type: :string,
-        desc: 'Which formatter to use: cli, progress, documentation, json, json-min, junit'
+        desc: '[DEPRECATED] Please use --format - this will be removed in InSpec 2.0'
```

Comment:
typo: `--format` is deprecated in favor of `--reporter`

Reply:
Thanks!